frontend/pages/index.tsx

```diff
@@ -13,14 +14,132 @@ import {
   imageSx,
   primaryTranslucentOverlaySx,
   secondaryTranslucentOverlaySx,
+  spacing2,
+  marTop5,
+  marTop10,
+  marLeft10,
+  marRight10,
+  largeFont,
+  impactText,
+  gradientSection,
+  buttonSecondary,
 } from '../styles/indexStyles';
+import Image from 'next/image';
 
 const Home: NextPage = () => {
   return (
     <div>
       <Head>
-        <title>Home</title>
+        <title>Giving Coupons</title>
       </Head>
+      <Box sx={hero}>
+        <Stack direction="column" justifyContent="flex-start" alignItems="center" spacing={2}>
+          <div style={{ display: 'block', height: '400px', width: '400px' }}>
+            <Image src="/nusstudents.png" layout="responsive" width="400px" height="400px"></Image>
+          </div>
+          <Typography variant="h1" align="center">
+            Together, we are more
+          </Typography>
+          <Typography sx={marTop5} variant="h3" align="center">
+            Giving coupons is an initiative to raise money for charities through public gift matching
+          </Typography>
+          <Button sx={buttonSx} actionType="primary">
+            Explore Our Campaigns
+          </Button>
+        </Stack>
+      </Box>
+
+      <Box sx={spacing2}>
+        <Typography sx={marTop10} variant="h1" align="center">
+          To Date
+        </Typography>
+        <Stack sx={spacing2} direction="column" justifyContent="space-between" alignItems="center" spacing={2}>
```

Comment:
Let's name the styles based on what component they're used for. This is for clarity. For example if this is the style for a section it can be `sectionSx`, or a style for the instruction points under how it works can be `instructionPointSx`